lib/phlex/component.rb

```diff
@@ -96,6 +96,8 @@ def _attributes(attributes)
           buffer << " " << k.name.tr("_", "-") << '="' << CGI.escape_html(v) << '"'
         when Symbol
           buffer << " " << k.name.tr("_", "-") << '="' << CGI.escape_html(v.name) << '"'
+        when Hash
+          _attributes(v.transform_keys { "#{k}-#{_1}".to_sym })
```

Comment:
Some of the stuff in `_attributes` doesn't need to be repeated for each nested attribute. We should probably extract another method, though I’m not sure what to call it.